integration/storagetest/maptestrunner.go

```diff
@@ -38,32 +38,35 @@ func RunMapStorageTests(t *testing.T, storageFactory MapStorageFactory) {
 	}
 }
 
-func mapTestFunctions(t *testing.T, x interface{}) map[string]MapStorageTest {
+func testFunctions(x interface{}) []string {
```

Comment:
Might be worth pulling this func out into its own file as it's shared between `maptestrunner` and `logtestrunner`?